pkg/backup/writer/abs_writer.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2017 The etcd-operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package writer
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"io"
+
+	"github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/coreos/etcd-operator/pkg/backup/util"
+	"github.com/pborman/uuid"
+)
+
+var _ Writer = &absWriter{}
+
+type absWriter struct {
+	abs *storage.BlobStorageClient
+}
+
+// NewABSWriter creates a abs writer.
+func NewABSWriter(abs *storage.BlobStorageClient) Writer {
+	return &absWriter{abs}
+}
+
+const (
+	// AzureBlobBlockChunkLimitInBytes 100MiB is the limit
+	AzureBlobBlockChunkLimitInBytes = 104857600
```

Comment:
prefer `AzureBlobBlockChunkLimitInBytes = 100 * 1024 * 1024` for readability; It is pretty clear this is `100MiB`